src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -1072,10 +1180,120 @@ inline void rpp_store48_f32pln3_to_f16pkd3_avx(Rpp16f *dstPtr, __m256 *p)
     _mm_storeu_si128((__m128i *)(dstPtr + 45), px128[3]);
 }
 
+inline void rpp_load96_u8pln3_to_f32pln3_avx512(Rpp8u *srcPtrR, Rpp8u *srcPtrG, Rpp8u *srcPtrB, __m512 *p)
+{
+    __m512i px[6];
+    px[0] = _mm512_loadu_si512((__m512i *)srcPtrR);
+    px[1] = _mm512_loadu_si512((__m512i *)srcPtrG);
+    px[2] = _mm512_loadu_si512((__m512i *)srcPtrB);
+    px[3] = _mm512_loadu_si512((__m512i *)(srcPtrR + 48));
+    px[4] = _mm512_loadu_si512((__m512i *)(srcPtrG + 48));
+    px[5] = _mm512_loadu_si512((__m512i *)(srcPtrB + 48));
+
+    __m128i input[4];
+    __mmask16 k = 0;
+    input[0] = _mm512_extracti32x4_epi32(px[0], 0);
+    input[1] = _mm512_extracti32x4_epi32(px[0], 1);
+    input[2] = _mm512_extracti32x4_epi32(px[0], 2);
+    input[3] = _mm512_extracti32x4_epi32(px[0], 3);
+
+    __m512i out[4];
+    out[0] = _mm512_cvtepu8_epi32 (input[0]);
+    out[1] = _mm512_cvtepu8_epi32 (input[1]);
+    out[2] = _mm512_cvtepu8_epi32 (input[2]);
+    out[3] = _mm512_cvtepu8_epi32 (input[3]);
+
+    __m512i output[3];
+    output[0] = _mm512_mask_blend_epi32 (k, out[0], out[1]);
```

Comment:
Please add comments for each instruction with the register level storage, so that it is easy to understand